README.md

```diff
@@ -90,11 +94,18 @@ challenge = client.challenge_from_hash(JSON.parse(File.read('challenge')))
 
 # Once you are ready to serve the confirmation request you can proceed.
 challenge.request_verification # => true
+authorization.verify_status # => 'pending'
 challenge.verify_status # => 'pending'
 
 # Wait a bit for the server to make the request, or just blink. It should be fast.
 sleep(1)
 
+# Rely on authorization.verify_status more than on challenge.verify_status,
+# if the former is 'valid' you can already issue a certificate and the status of
+# the challenge is not relevant and in fact may never change from pending.
+# However if authorization.verify_status is 'invalid', you can get at the error
+# message only through the challenge.
+authorization.verify_status # => 'valid'
```

Comment:
Well, when `client.authorize(...).status` isn't `valid` you're going to submit a challenge and that challenge will be processed because the authorization wasn't `valid`, so for some code designs not keeping the authorization around but just the challenge can work and be valid usage.